lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_tableaus.jl

```diff
@@ -339,167 +339,43 @@ function Rodas4P2Tableau(T, T2)
-    C41 = convert(T, 32.22751541853323)
-    C42 = convert(T, -4.943732386540191)
-    C43 = convert(T, 19.44922031041879)
-    C51 = convert(T, -20.69865579590063)
-    C52 = convert(T, -8.816374604402768)
-    C53 = convert(T, 1.260436877740897)
-    C54 = convert(T, -0.7495647613787146)
-    C61 = convert(T, -46.22004352711257)
-    C62 = convert(T, -17.49534862857472)
-    C63 = convert(T, -289.6389582892057)
-    C64 = convert(T, 93.60855400400906)
-    C65 = convert(T, 318.3822534212147)
-    C71 = convert(T, 34.20013733472935)
-    C72 = convert(T, -14.15535402717690)
-    C73 = convert(T, 57.82335640988400)
-    C74 = convert(T, 25.83362985412365)
-    C75 = convert(T, 1.408950972071624)
-    C76 = convert(T, -6.551835421242162)
-    C81 = convert(T, 42.57076742291101)
-    C82 = convert(T, -13.80770672017997)
-    C83 = convert(T, 93.98938432427124)
-    C84 = convert(T, 18.77919633714503)
-    C85 = convert(T, -31.58359187223370)
-    C86 = convert(T, -6.685968952921985)
-    C87 = convert(T, -5.810979938412932)
-    c2 = convert(T2, 0.38)
-    c3 = convert(T2, 0.3878509998321533)
-    c4 = convert(T2, 0.4839718937873840)
-    c5 = convert(T2, 0.4570477008819580)
-    d1 = convert(T, gamma)
-    d2 = convert(T, -0.1823079225333714636)
-    d3 = convert(T, -0.319231832186874912)
-    d4 = convert(T, 0.3449828624725343)
-    d5 = convert(T, -0.377417564392089818)
-
-    h21 = convert(T, 27.354592673333357)
-    h22 = convert(T, -6.925207756232857)
-    h23 = convert(T, 26.40037733258859)
-    h24 = convert(T, 0.5635230501052979)
-    h25 = convert(T, -4.699151156849391)
-    h26 = convert(T, -1.6008677469422725)
-    h27 = convert(T, -1.5306074446748028)
-    h28 = convert(T, -1.3929872940716344)
-
-    h31 = convert(T, 44.19024239501722)
-    h32 = convert(T, 1.3677947663381929e-13)
-    h33 = convert(T, 202.93261852171622)
-    h34 = convert(T, -35.5669339789154)
-    h35 = convert(T, -181.91095152160645)
-    h36 = convert(T, 3.4116351403665033)
-    h37 = convert(T, 2.5793540257308067)
-    h38 = convert(T, 2.2435122582734066)
-
-    h41 = convert(T, -44.0988150021747)
-    h42 = convert(T, -5.755396159656812e-13)
-    h43 = convert(T, -181.26175034586677)
-    h44 = convert(T, 56.99302194811676)
-    h45 = convert(T, 183.21182741427398)
-    h46 = convert(T, -7.480257918273637)
-    h47 = convert(T, -5.792426076169686)
-    h48 = convert(T, -5.32503859794143)
-
+    A = T[
+        0                  0                 0                   0                  0 0
+        2.0                0                 0                   0                  0 0
+        3.040894194418781  1.041747909077569 0                   0                  0 0
+        2.576417536461461  1.622083060776640 -0.9089668560264532 0                  0 0
+        2.760842080225597  1.446624659844071 -0.3036980084553738 0.2877498600325443 0 0
+        -14.09640773051259 6.925207756232704 -41.47510893210728  2.343771018586405  24.13215229196062 0
```

Comment:
This needs an extra row of `-14.09640773051259 6.925207756232704 -41.47510893210728 2.343771018586405 24.13215229196062 1 0` (and the rest of the matrix needs an extra column of `0`)

Reply:
Extra row still missing.